tests/integration/Mvc/Model/BinderCest.php

```diff
@@ -0,0 +1,500 @@
+
+        $returnedValue = $dispatcher->getReturnedValue();
+        $I->assertInstanceOf('Phalcon\Test\Models\People', $returnedValue);
+        $I->assertEquals($people->cedula, $returnedValue->cedula);
+        $I->assertEquals([
+            'people' => 'Phalcon\\Test\\Models\\People',
+        ], $this->cache->get('_PHMB_Test11Controller_viewAction'));
+
+        $dispatcher->setActionName('multiple');
+        $dispatcher->setParams(['people' => $people->cedula, 'robots' => $robot->id]);
+
+        $dispatcher->dispatch();
+
+        $returnedValue = $dispatcher->getReturnedValue();
+        $I->assertInstanceOf('Phalcon\Test\Models\People', $returnedValue[0]);
+        $I->assertInstanceOf('Phalcon\Test\Models\Robots', $returnedValue[1]);
+        $I->assertEquals($people->cedula, $returnedValue[0]->cedula);
+        $I->assertEquals($robot->id, $returnedValue[1]->id);
+        $I->assertEquals([
+            'people' => 'Phalcon\\Test\\Models\\People',
+            'robots' => 'Phalcon\\Test\\Models\\Robots',
+        ], $this->cache->get('_PHMB_Test11Controller_multipleAction'));
+
+        // Reset dispatcher
+        $dispatcher = new Dispatcher;
+        $dispatcher->setDI(Di::getDefault());
+
+        $I->assertInstanceOf('Phalcon\Di', $dispatcher->getDI());
+        $I->haveServiceInDi('dispatcher', $dispatcher);
+
+        $dispatcher->setControllerName('test9');
+        $dispatcher->setActionName('view');
+        $dispatcher->setParams(['people' => $people->cedula]);
+
+        try {
+            $dispatcher->dispatch();
+
+            $I->assertTrue(
+                false,
+                'Here must be the exception about passing non model to the controller action'
+            );
+        } catch (\Exception $e) {
+            // PHP 5.x
+            $I->assertEquals(
+                'Argument 1 passed to Test9Controller::viewAction() must be an instance of Phalcon\Mvc\Model, string given',
+                $e->getMessage()
+            );
+        } catch (\TypeError $e) {
+            // PHP 7.x
+            $I->assertEquals(
+                'Argument 1 passed to Test9Controller::viewAction() must be an instance of Phalcon\Mvc\Model, string given',
+                $e->getMessage()
+            );
+        }
+
+        $dispatcher->setModelBinder($this->modelBinder);
+        $dispatcher->dispatch();
+
+        $returnedValue = $dispatcher->getReturnedValue();
+        $I->assertInstanceOf('Phalcon\Test\Models\People', $returnedValue);
+        $I->assertEquals($returnedValue->cedula, $people->cedula);
+        $I->assertEquals([
+            'people' => 'Phalcon\\Test\\Models\\People',
+        ], $this->cache->get('_PHMB_Test9Controller_viewAction'));
+    }
+
+    /**
+     * Tests micro with handlers and model binder
+     *
+     * @author Wojciech Ślawski <jurigag@gmail.com>
+     * @since  2016-10-20
+     *
+     * @param IntegrationTester $I
+     */
+    public function testMicroHandler(IntegrationTester $I)
```

Comment:
`public function testMicroHandlerAndModelBinder`

anyway I think you can make the test name even more verbose and specific